COCO_DATA_FORMAT.md

```diff
@@ -110,6 +110,21 @@ Here is one example of the json file for the image regression task, where the "t
 }
 ```
 
+## Image retrieval
+
+Here is one example of the json file for the image retrieval task. The image retrieval json file is similar to image_caption and image text matching dataset formats in that it contains text associated to images. However, there are some important differences: 1) There is an optional category section which allows the definition of both a category name and a super category. 2) Unlike the other 2 dataset formats, the annotations contain a "query" field rather than "text" or "caption" to match the task at hand.
+
+``` {json}
+{
+    "images":[{"id":1,"zip_file":"test1.zip","file_name":"test/0/image_1.jpg"},
+              {"id":2,"zip_file":"test2.zip","file_name":"test/1/image_2.jpg"}],
+    "categories":[{"id":1,"name":"white","supercategory":"race"},
```

Comment:
should it be super_category to keep the naming consistent?

Reply:
for this one, I used the naming from here: https://cocodataset.org/#format-data

Reply:
sounds good.